packages/gatsby-plugin-offline/src/sw-append.js

```diff
@@ -1,29 +1 @@
-/* global workbox */
-
-self.addEventListener(`message`, event => {
-  const { api } = event.data
-  if (api === `gatsby-runtime-cache`) {
-    const { resources } = event.data
-    const cacheName = workbox.core.cacheNames.runtime
-
-    event.waitUntil(
-      caches.open(cacheName).then(cache =>
-        Promise.all(
-          resources.map(resource => {
-            let request
-
-            // Some external resources don't allow
-            // CORS so get an opaque response
-            if (resource.match(/^https?:/)) {
-              request = fetch(resource, { mode: `no-cors` })
-            } else {
-              request = fetch(resource)
-            }
-
-            return request.then(response => cache.put(resource, response))
-          })
-        )
-      )
-    )
-  }
-})
+// noop
```

Comment:
If we're no longer using this, do we even need this file at all?

Additionally, we probably can remove the appending stuff, as well.

https://github.com/gatsbyjs/gatsby/blob/e02f9d54c05721fd96a8597954fc844d31386b8e/packages/gatsby-plugin-offline/src/gatsby-node.js#L125